### DISCUSSION TITLE
Karavan vscode Extension

### DISCUSSION THREAD

**Discussion by ravishankar-hassain** [apache/camel-karavan#1051] [General]
Is there an option to deploy the Karavan application into the namespace of our choice in the Kubernetes cluster which is running in the public cloud / on-prem by providing the option

**Comment by mgubaidullin** [comment]
![deploy-namespace](https://github.com/apache/camel-karavan/assets/1379213/8e90b350-2d1f-40dd-9e1c-8e072dc475cd)

**Reply by ravishankar-hassain** [reply]
Thanks for the quick revert. In the above demo I guess minikube is used as the Kubernetes cluster. Could you please kindly assist on how to deploy the Karavan App to Kubernetes Cluster which is running either in public cloud / on-prem by providing the option

**Comment by mgubaidullin** [comment]
Same way. Just configure your kubectl to use your cluster

**Comment by ravishankar-hassain** [comment]
> Same way. Just configure your kubectl to use your cluster
Is it possible to configure this in Karavan  vs extension directly to log into the Kubernetes cluster of my choice instead and do the deployment ?

**Reply by mgubaidullin** [reply]
Karavan VSCode extension is an integration solution and not aimed to replace kubectl.
However you could install Karavan web application to your Kubernetes cluster and then Karavan will deploy directly into your Kubernetes cluster.
